ui/v2.5/src/components/Settings/SettingsScrapingPanel.tsx

```diff
@@ -26,13 +26,14 @@ import {
 import { ExternalLink } from "../Shared/ExternalLink";
 import { ClearableInput } from "../Shared/ClearableInput";
 import { Counter } from "../Shared/Counter";
+import { PatchComponent } from "src/patch"; // Import PatchComponent
 
 const ScraperTable: React.FC<
   PropsWithChildren<{
     entityType: string;
     count?: number;
   }>
-> = ({ entityType, count, children }) => {
+> = PatchComponent("ScraperTable", ({ entityType, count, children }) => {
```

Comment:
As above, I think this should be SettingsScrapingPanel.ScraperTable. I won't mark all of these, but please do a pass and correct the non-exported ones.